README.md

```diff
@@ -49,13 +49,19 @@ For **Integration Edition and Document Server integrated with a 3rd-party storag
         ...
     });
     ```
+**Important**: when you integrate ONLYOFFICE Document Server with a 3rd-party storage, you need to use special connectors (integration apps). If you compile a connector from source code or create a new one, you can add plugins using Document Server config. If you use ready connectors (e.g. from ownCloud/Nextcloud marketplaces) adding plugins via config is not applicable.
```

Comment:
> If you use ready connectors (e.g. from ownCloud/Nextcloud marketplaces)

does zotero plugin aviable from some kind of marketplace? If not - maybe drop this line alltogether?

Reply:
We are talking about connectors for ownCloud/Nextcloud and so on here, not Zotero plugin itself. 
Adding plugins using config won't work for our ready connectors, because users can't change them (NC connector is signed, for example). If that's not clear, maybe I should change this part

Reply:
Ah, ok got it